LongWarOfTheChosen/Src/LW_PerkPack_Integrated/Classes/X2Ability_PerkPackAbilitySet2.uc

```diff
@@ -412,7 +418,11 @@ static function X2AbilityTemplate AddSmokeGrenade()
 	Template.Hostility = eHostility_Neutral;
 	Template.AbilityToHitCalc = default.DeadEye;
 	Template.AbilityTargetStyle = default.SelfTarget;
-	Template.AbilityTriggers.AddItem(default.UnitPostBeginPlayTrigger);
+
+	Trigger = new class'X2AbilityTrigger_UnitPostBeginPlay';
+	Trigger.Priority -= 20; // delayed so that Full Kit happen first
```

Comment:
Hold on, I'm pretty sure lower priority = goes faster, you sure you tested it?

Reply:
I believe that I did test it at the time. 
For example here is the line form Dense Smoke ability:
`Trigger.Priority -= 40; // delayed so that any other abilities that add items happen first`
Or this line from vanilla HotLoadAmmo():
`PostBeginPlayTrigger.Priority -= 10;        // Lower priority to guarantee ammo modifying effects (e.g. Deep Pockets) already run.`
So I believe higher priority should happen earlier, as the name suggests.